apollo-ios/Sources/Apollo/ApolloStore.swift

```diff
@@ -83,13 +85,23 @@ public class ApolloStore {
     }
   }
 
-  func subscribe(_ subscriber: ApolloStoreSubscriber) {
+  /// Subscribes to notifications of ApolloStore content changes
+  ///
+  /// - Parameters:
+  ///    - subscriber: A subscriber to receive content change notificatons. To avoid a retain cycle,
+  ///                  ensure you call `unsubscribe` on this subcriber before it goes out of scope.
```

Comment:
```suggestion
  ///                  ensure you call `unsubscribe` on this subscriber before it goes out of scope.
```

Reply:
This fix ended up in https://github.com/apollographql/apollo-ios-dev/pull/300/commits/6f66f7be3dcce250c43c20b11bde2b53df2b615f